src/go/k8s/config/manager/manager.yaml

```diff
@@ -29,7 +29,7 @@ spec:
         - /manager
         args:
         - --leader-elect
-        image: controller:latest
+        image: vectorized/redpanda-operator:2021022-e853bf3
```

Comment:
I am fine with this temporarily but let's do semver

Reply:
I'm open for discussion, but not in this PR :)

Reply:
Without starting a discussion around semver, do you already have a plan in mind for changing or updating this version, or is it open?